content/pt/docs/what-is-opentelemetry.md

```diff
@@ -0,0 +1,133 @@
+No contexto de software, isso significa ser capaz de entender o estado interno
+do sistema analisando seus dados de telemetria, que incluem rastros, métricas e
+logs.
+
+Para tornar um sistema observável, ele deve ser
+[instrumentado](/docs/concepts/instrumentation). Ou seja, o código deve emitir
+[rastros](/docs/concepts/signals/traces/),
+[métricas](/docs/concepts/signals/metrics/) ou
+[logs](/docs/concepts/signals/logs/). Os dados instrumentados devem então ser
+enviados para um backend de observabilidade.
+
+## Por que OpenTelemetry?
+
+Com a ascensão da computação em nuvem, arquiteturas de microsserviços e dos
+requisitos de negócios cada vez mais complexos, a necessidade de
+[observabilidade](/docs/concepts/observability-primer/#what-is-observability) de
+software e infraestrutura é cada vez maior.
+
+O OpenTelemetry atende à necessidade de observabilidade seguindo dois
+princípios-chave:
+
+1. Você é o proprietário dos dados que gera. Não há dependência de fornecedor
+   (_lock-in_).
+2. Você só precisa aprender um único conjunto de APIs e convenções.
+
+Ambos os princípios combinados concedem às equipes e organizações a
+flexibilidade de que precisam no mundo da computação moderna de hoje.
+
+Se você quiser saber mais, dê uma olhada na
+[missão, visão e valores](/community/mission/) do OpenTelemetry.
+
+## Os principais componentes do OpenTelemetry
+
+OpenTelemetry consiste dos seguintes componentes:
+
+- Uma [especificação](/docs/specs/otel) para todos os componentes
+- Um [protocolo](/docs/specs/otlp/) padrão que defini a forma dos dados de
+  telemetria
+- [Convenções semânticas](/docs/specs/semconv/) que estabelecem um padrão de
+  nomenclatura para tipos comuns de dados de telemetria.
+- APIs que define como gerar dados de telemetria
+- [SDKs de linguagem](/docs/languages) que implementam a especificação, APIs e
+  exportação de dados de telemetria
+- Um [ecossistema de bibliotecas](/ecosystem/registry) que implementa
+  instrumentação para bibliotecas e estruturas de trabalho comuns
+- Componentes de instrumentação automática que geram dados de telemetria sem
+  exigir alterações no código
+- O [OpenTelemetry Collector](/docs/collector), é um intermediário (_proxy_) que
+  recebe, processa e exporta dados de telemetria
+- Várias outras ferramentas, como o
+  [OpenTelemetry Operador para Kubernetes](/docs/kubernetes/operator/),
+  [OpenTelemetry Helm Charts](/docs/kubernetes/helm/), e
+  [comunidade ativa para FaaS](/docs/faas/)
+
+O OpenTelemetry é utilizado por uma grande variedade de
+[bibliotecas, serviços e aplicativos](/ecosystem/integrations/) que o integram
+para fornecer um padrão de observabilidade.
+
+OpenTelemetry é suportado por inúmeros [fornecedores](/ecosystem/vendors/),
+muitos dos quais oferecem suporte comercial para OpenTelemetry e contribuem
+diretamente para o projeto.
+
+## Extensibilidade
+
+O OpenTelemetry é projetado para ser extensível. Alguns exemplos de como ele
```

Comment:
Sad that there's no gender neutral way to address "OpenTelemetry". :(

Reply:
I don't like this sentence, too 😢. 
@joaopgrassi What do you think about `O OpenTelemetry é projetado para ser flexível.`?

Reply:
Ah, the sentence is good to me, what is sad is that we refer to OTel as "he" because I guess there isn't any neutral way as we have in English. 

Perhaps we can say:

> OpenTelemetry é projetado para ser extensível. Alguns exemplos de como o projeto... 

Then we avoid using "O, ele" and etc

Reply:
I think in this case I think it's OK since we're referring to a "thing" (the OpenTelemetry project). There was also a previous discussion about keeping the article for consistency in https://github.com/open-telemetry/opentelemetry.io/pull/4868#discussion_r1690339198.

Gender neutral phrasing should be applied when referring to people (specially the reader), such as in https://github.com/open-telemetry/opentelemetry.io/pull/4896#discussion_r1698742689